android/CMakeLists.txt

```diff
@@ -234,12 +205,7 @@ else()
         )
 endif()
 
-find_library(
-        REANIMATED_LIB
-        reanimated
-        PATHS ${LIBREANIMATED_DIR}
-        NO_CMAKE_FIND_ROOT_PATH
-)
+find_package(react-native-reanimated REQUIRED CONFIG)
```

Comment:
Now, RNVisionCamera consumes Reanimated as a prefab (both shared library and header files) 🚀

Reply:
amazing! build setup is so much cleaner now 🤩